engo_test.go

```diff
@@ -0,0 +1,46 @@
+package engo
+
+import (
+	"testing"
+
+	"github.com/github.com/stretchr/testify/assert"
+)
+
+func TestSetScaleOnResize(t *testing.T) {
```

Comment:
Testing that said function is doing what it's supposed to. If it did anything else, these will fill.